pset-iterators.js

```diff
@@ -115,6 +134,15 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+const presidentsBetween1850and1900 = presidents.filter(function (president) {
+  started=president.took_office.split(" ")
+  return parseInt(started) >= 1850 && parseInt(started) < 1900
+})
+
+console.log("Presidents who took office between 1850 and 1900:")
+console.log(presidentsBetween1850and1900)
+
+
```

Comment:
@AtleLohrmann Great work here using the filter method to return all the president who served between 1850 and 1900. Good Job using the parseInt to change the values from a string to a number.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L124)